exchanges/bitstamp/bitstamp_test.go

```diff
@@ -741,41 +733,34 @@ func TestWsUnsubscribe(t *testing.T) {
-	if err = b.wsHandleData(pressXToJSON); !errors.Is(err, errWSPairParsingError) {
-		t.Errorf("expected %s, got %s", errWSPairParsingError, err)
-	}
+	err = b.wsHandleData(pressXToJSON)
+	assert.Equal(t, errWSPairParsingError, err, "TestWsOrderbook should error parsing error")
```

Comment:
Just a side note, you don't need to convert over to asserting if you don't want to. As long as existing testing works it can be left as is, just to reduce work load.

Reply:
Couldn't resist changing it to maintain consistency :sweat_smile:, hope that's ok